Makefile

```diff
@@ -185,15 +185,15 @@ run: manifests generate fmt vet ## Run a controller from your host.
 .PHONY: docker-build
 docker-build:  ## Build docker image with the manager.
 	echo "Git commit hash: ${GIT_COMMIT_HASH}"
-	KO_DOCKER_REPO=ko.local GOFLAGS="-ldflags=-X=main.gitCommitHash=${GIT_COMMIT_HASH}" ko build -B --platform=${PLATFORMS} -t ${IMG_TAG} -L .
+	KO_DOCKER_REPO=ko.local GOFLAGS="-ldflags=-X=main.gitCommitHash=${GIT_COMMIT_HASH}" ko build -B --platform=${PLATFORMS} -t ${IMG_TAG} .
```

Comment:
why is the local flag removed?

Reply:
to remove following warning <pre>
 ✘  ~/go/src/github.com/nutanix-cloud-native/cluster-api-provider-nutanix   jira/krbn-8065  make docker-build-e2e
echo "Git commit hash: 00ef89b8f5368ca5011e8122c492fe885218f8ed"
Git commit hash: 00ef89b8f5368ca5011e8122c492fe885218f8ed
KO_DOCKER_REPO=ko.local GOFLAGS="-ldflags=-X=main.gitCommitHash=00ef89b8f5368ca5011e8122c492fe885218f8ed" ko build -B --platform=linux/amd64 -t e2e-00ef89b8f5368ca5011e8122c492fe885218f8ed -L .
2024/03/18 08:58:12 WARNING!
-----------------------------------------------------------------
The --local flag is set and KO_DOCKER_REPO is set to ko.local

You can choose either one to build a local image.

The --local flag might be deprecated in the future.
-----------------------------------------------------------------
</pre>